frame/contracts/src/benchmarking/mod.rs

```diff
@@ -1300,7 +1300,10 @@ benchmarks! {
 		let hashes = (0..r * API_BENCHMARK_BATCH_SIZE)
 			.map(|i| {
 				let code = WasmModule::<T>::from(ModuleDefinition {
+					memory: Some(ImportedMemory::max::<T>()),
 					call_body: Some(body::plain(vec![
+						// we need to add this in order to make contracts unique
+						// so that they can be deployed from the same sender
```

Comment:
:eyes:

Reply:
This is what I mean when I say we need to make assumptions about the address generation in #7482. Once that PR is merged I can remove this and also create a test from that benchmark using `create_test!`.